internal/c/libqb/src/glut-message.h

```diff
@@ -72,58 +70,54 @@ class glut_message_get : public glut_message {
     int response_value;
     void execute();
 
-    glut_message_get(int _id) : glut_message(true), id(_id), response_value(0) { }
+    glut_message_get(int _id) : glut_message(true), id(_id), response_value(0) {}
 };
 
 class glut_message_iconify_window : public glut_message {
   public:
     void execute();
 
-    glut_message_iconify_window() : glut_message(false) { }
+    glut_message_iconify_window() : glut_message(false) {}
 };
 
 class glut_message_position_window : public glut_message {
   public:
     int x, y;
     void execute();
 
-    glut_message_position_window(int _x, int _y) : glut_message(false), x(_x), y(_y) { }
+    glut_message_position_window(int _x, int _y) : glut_message(false), x(_x), y(_y) {}
 };
 
 class glut_message_show_window : public glut_message {
   public:
     void execute();
 
-    glut_message_show_window() : glut_message(false) { }
+    glut_message_show_window() : glut_message(false) {}
 };
 
 class glut_message_hide_window : public glut_message {
   public:
     void execute();
 
-    glut_message_hide_window() : glut_message(false) { }
+    glut_message_hide_window() : glut_message(false) {}
 };
 
 class glut_message_set_window_title : public glut_message {
   public:
     char *newTitle;
     void execute();
 
-    glut_message_set_window_title(const char *title) : glut_message(false) {
-        newTitle = strdup(title);
-    }
+    glut_message_set_window_title(const char *title) : glut_message(false) { newTitle = strdup(title); }
```

Comment:
Maybe we don't have to put `AllowShortFunctionsOnASingleLine` on :D IMO some of these one-line functions are too long for a single line.

Reply:
Thank you. I always found that weird to be honest. Changed it to `Empty` so that only empty functions get to be on the same line. Also added `SeparateDefinitionBlocks: Always` so that we always add an empty line after classes, structs, enums, and functions.